master/models.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"time"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+// User model prototype without hashing algorithms implemented yet
+type User struct {
+	ID               bson.ObjectId `bson:"_id,omitempty"`
+	Username         string        `bson:"username,omitempty"`
```

Comment:
Use Username as the id field of a User MongoDB Document (to enforce username uniqueness).